### ISSUE TITLE
Feature Request: Be able to combine same process names in process list.

### ISSUE THREAD

**Issue by Jahhein** [aksakalli/gtop#23]
#### Description

Putting in a request for a feature to be added to the package. The ability to combine processes by name in the process list. Example: Google Chrome Helper on OS X. There are many processes that contain the same name and it'd be easier to see what is hogging CPU/Memory if combined.

Great package!

**Comment by mirfilip** [comment]
Of course, the optimal would be to have that as an option, not a default.

**Comment by Jahhein** [comment]
Awesome! Sounds good.

Just also thought that perhaps adding a process count as well as a default for repeated processes.
